tests/installer/installer_efi_test.go

```diff
@@ -220,7 +220,7 @@ var _ = Describe("cOS Installer EFI tests", func() {
 			})
 			It("forces efi", func() {
 				By("Running the installer")
-				out, err := s.Command("elemental install --force-efi /dev/sda")
+				out, err := s.Command(fmt.Sprintf("%s install --force-efi /dev/sda", sut.ElementalCmd))
```

Comment:
mmm maybe we could even do an `s.Elemental` helper directly?

Reply:
yep, that should be the endgame, `s.Elemental(arg1, arg2, arg3)`